products/llbuildSwift/BuildSystemBindings.swift

```diff
@@ -488,12 +513,20 @@ public final class BuildSystem {
         llb_buildsystem_destroy(_system)
     }
 
-    /// Build the default target.
+    /// Build the default target, or optionally a single node.
     ///
     /// The client is responsible for ensuring only one build is ever executing concurrently.
-    public func build() -> Bool {
-        var data = llb_data_t(length: 0, data: nil)
-        return llb_buildsystem_build(_system, &data)
+    ///
+    /// - parameter node: Optional path to a single node to build.
+    /// - returns: True if the build was successful, false otherwise.
+    public func build(node: String? = nil) -> Bool {
+        if let node = node {
+            var data = copiedDataFromBytes([UInt8](node.utf8))
+            return llb_buildsystem_build_node(_system, &data)
```

Comment:
Can this be simplified to `copiedDataFromBytes(node.map{ [UInt8]($0.utf8) } ?? [])` and eliminate the if?

Reply:
Don't think so. Its calling a different method based on presence of a node.